contents/blog/how-to-fix-your-growth-team.md

```diff
@@ -0,0 +1,133 @@
+
+The best way to do this is to create a bottom-up culture where it's the job of **every single person** to come up with ideas. That means every engineer, designer, data scientist, sales rep, marketer, etc.
+
+When it's everyone's job to do this, the number of ideas increases dramatically. Then, by applying the EVENLYN framework mentioned above, you can distill the list to only the best ideas.
+
+> **💡 Tip:** To create an inclusive ideas culture, make your [brainstorms asynchronous](https://source.opennews.org/articles/redesigning-brainstorming-asynchronous/).
+
+### Maximize the impact of every experiment you ship
+
+Every engineer must own the experiments they are working on. They are the ones responsible for making them as impactful as possible – even if it wasn't originally their idea! 
+
+In practice, this means they should work to improve the initial hypothesis and develop further iterations to test. This also means allowing them to fail without being overly harsh on them.
+
+## Problem 4: Your team is afraid of making mistakes
+
+If your team is not making mistakes, you're not moving fast enough. 
+
+In fact, it's not uncommon for growth teams to cause major incidents! And their goal must not be to avoid them.
+
+"There's an anecdote that when airbags were introduced in cars, the average speed on the highway went up. This is essentially what you want to do for your growth team. Give them the freedom to make mistakes, but make sure they resolve them quickly."
+
+### How to fix it
+
+Your aim should be to build a resilient system that can rapidly recover from mistakes or incidents.
+
+In the context of growth teams, there are a few ways to do this:
+
+- Monitor critical metrics and set up alerts. Ensure you can quickly roll back changes using [configurators or feature flags](https://posthog.com/blog/feature-flags-vs-configuration).
+- Set up [review apps](https://seanconnolly.dev/review-applications) so your team can easily test and dogfood your experiments.
+- To test releases in production, set up [conditional overrides](/docs/experiments/testing-and-launching) so you can quickly test each experiment variant.
+- Instead of crashing your app when rendering errors occur, display a fallback UI. In React, you can easily add these with [Error Boundaries](https://react.dev/reference/react/Component#catching-rendering-errors-with-an-error-boundary) (and similar concepts existing in other frameworks).
+
+Notice that none of the solutions involve writing tests, which are a **waste of time** for experiments that may only last for a week.
+
+## Final tip: Your team may be tired
+
+It's okay if your engineers are tired. Growth engineering may not be as fulfilling for some people, especially if they yearn to build deep and complex systems.
+
+It's common for engineers to start feeling bored after a few quarters, and there's no shame in it. A symptom of this is diminishing returns from the team's work. 
```

Comment:
not necessarily bored - uneasy, anxious, malaise, etc.

Reply:
I think "unsatisfied" sums it up nicely